src/Deferred.php

```diff
@@ -19,33 +19,31 @@
 
 namespace Later;
 
+use Throwable;
+
 /**
  * Deferred: a wrapper object.
  *
  * @template T
  *
  * @template-implements Interfaces\Deferred<T>
  *
- * @psalm-suppress PropertyNotSetInConstructor
- *
  * @internal
+ * @final
  */
-final class Deferred implements Interfaces\Deferred
+class Deferred implements Interfaces\Deferred
 {
     /**
      * @var ?iterable<T>
```

Comment:
Please write `iterable<T>|null`

Reply:
How can we argue this will not contradict the other two instances below that use `?type` notation?

Reply:
All right.
You've won 🏆